Build failure: python313FreeThreading.pkgs.cython

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

run `nix build nixpkgs#python313FreeThreading.pkgs.cython`

### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
this derivation will be built:
  /nix/store/m9xm38wch7fjrmwvia27nl60xdzm3j98-python3.13t-cython-3.0.12.drv
building '/nix/store/m9xm38wch7fjrmwvia27nl60xdzm3j98-python3.13t-cython-3.0.12.drv'...
python3.13t-cython> bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8): No such file or directory
python3.13t-cython> Sourcing python-remove-tests-dir-hook
python3.13t-cython> Sourcing python-catch-conflicts-hook.sh
python3.13t-cython> Sourcing python-remove-bin-bytecode-hook.sh
python3.13t-cython> Sourcing pypa-build-hook
python3.13t-cython> Using pypaBuildPhase
python3.13t-cython> Sourcing python-runtime-deps-check-hook
python3.13t-cython> Using pythonRuntimeDepsCheckHook
python3.13t-cython> Sourcing pypa-install-hook
python3.13t-cython> Using pypaInstallPhase
python3.13t-cython> Sourcing python-imports-check-hook.sh
python3.13t-cython> Using pythonImportsCheckPhase
python3.13t-cython> Sourcing python-namespaces-hook
python3.13t-cython> Sourcing python-catch-conflicts-hook.sh
python3.13t-cython> Running phase: unpackPhase
python3.13t-cython> unpacking source archive /nix/store/gl3cxnhh8b2924sbxx8lfr27bjlhlba5-source
python3.13t-cython> source root is source
python3.13t-cython> setting SOURCE_DATE_EPOCH to timestamp 315619200 of file "source/tox.ini"
python3.13t-cython> Running phase: patchPhase
python3.13t-cython> Running phase: updateAutotoolsGnuConfigScriptsPhase
python3.13t-cython> Running phase: configurePhase
python3.13t-cython> no configure script, doing nothing
python3.13t-cython> Running phase: buildPhase
python3.13t-cython> Executing pypaBuildPhase
python3.13t-cython> Creating a wheel...
python3.13t-cython> pypa build flags: --no-isolation --outdir dist/ --wheel
python3.13t-cython> /nix/store/xy4jjgw87sbgwylm5kn047d9gkbhsr9x-bash-5.2p37/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8): No such file or directory
python3.13t-cython> * Getting build dependencies for wheel...
python3.13t-cython> <string>:118: DeprecationWarning: Use shutil.which instead of find_executable
python3.13t-cython> Unable to find pgen, not compiling formal grammar.
python3.13t-cython> /nix/store/3911v0wkrq9wc3d311y57y1bynjjzmad-python3.13t-setuptools-78.1.0/lib/python3.13t/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
python3.13t-cython> !!
python3.13t-cython>
python3.13t-cython>         ********************************************************************************
python3.13t-cython>         Please consider removing the following classifiers in favor of a SPDX license expression:
python3.13t-cython>
python3.13t-cython>         License :: OSI Approved :: Apache Software License
python3.13t-cython>
python3.13t-cython>         See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
python3.13t-cython>         ********************************************************************************
python3.13t-cython>
python3.13t-cython> !!
python3.13t-cython>   self._finalize_license_expression()
python3.13t-cython> running egg_info
python3.13t-cython> creating Cython.egg-info
python3.13t-cython> writing Cython.egg-info/PKG-INFO
python3.13t-cython> writing dependency_links to Cython.egg-info/dependency_links.txt
python3.13t-cython> writing entry points to Cython.egg-info/entry_points.txt
python3.13t-cython> writing top-level names to Cython.egg-info/top_level.txt
python3.13t-cython> writing manifest file 'Cython.egg-info/SOURCES.txt'
python3.13t-cython> reading manifest file 'Cython.egg-info/SOURCES.txt'
python3.13t-cython> reading manifest template 'MANIFEST.in'
python3.13t-cython> warning: no files found matching '.gitrev'
python3.13t-cython> warning: no files found matching 'Doc/*'
python3.13t-cython> warning: no files found matching '*.pyx' under directory 'Cython/Debugger/Tests'
python3.13t-cython> warning: no files found matching '*.pxd' under directory 'Cython/Debugger/Tests'
python3.13t-cython> warning: no files found matching '*.pxd' under directory 'Cython/Utility'
python3.13t-cython> warning: no files found matching 'pyximport/README'
python3.13t-cython> adding license file 'LICENSE.txt'
python3.13t-cython> adding license file 'COPYING.txt'
python3.13t-cython> writing manifest file 'Cython.egg-info/SOURCES.txt'
python3.13t-cython> * Building wheel...
python3.13t-cython> <string>:118: DeprecationWarning: Use shutil.which instead of find_executable
python3.13t-cython> Unable to find pgen, not compiling formal grammar.
python3.13t-cython> /nix/store/3911v0wkrq9wc3d311y57y1bynjjzmad-python3.13t-setuptools-78.1.0/lib/python3.13t/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
python3.13t-cython> !!
python3.13t-cython>
python3.13t-cython>         ********************************************************************************
python3.13t-cython>         Please consider removing the following classifiers in favor of a SPDX license expression:
python3.13t-cython>
python3.13t-cython>         License :: OSI Approved :: Apache Software License
python3.13t-cython>
python3.13t-cython>         See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
python3.13t-cython>         ********************************************************************************
python3.13t-cython>
python3.13t-cython> !!
python3.13t-cython>   self._finalize_license_expression()
python3.13t-cython> running bdist_wheel
python3.13t-cython> /nix/store/3911v0wkrq9wc3d311y57y1bynjjzmad-python3.13t-setuptools-78.1.0/lib/python3.13t/site-packages/setuptools/_distutils/cmd.py:135: SetuptoolsDeprecationWarning: bdist_wheel.universal is deprecated
python3.13t-cython> !!
python3.13t-cython>
python3.13t-cython>         ********************************************************************************
python3.13t-cython>         With Python 2.7 end-of-life, support for building universal wheels
python3.13t-cython>         (i.e., wheels that support both Python 2 and Python 3)
python3.13t-cython>         is being obviated.
python3.13t-cython>         Please discontinue using this option, or if you still need it,
python3.13t-cython>         file an issue with pypa/setuptools describing your use case.
python3.13t-cython>
python3.13t-cython>         By 2025-Aug-30, you need to update your project and remove deprecated calls
python3.13t-cython>         or your builds will no longer be supported.
python3.13t-cython>         ********************************************************************************
python3.13t-cython>
python3.13t-cython> !!
python3.13t-cython>   self.finalize_options()
python3.13t-cython> running build
python3.13t-cython> running build_py
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t
python3.13t-cython> copying cython.py -> build/lib.linux-x86_64-cpython-313t
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> copying Cython/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> copying Cython/Utils.py -> build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> copying Cython/TestUtils.py -> build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> copying Cython/StringIOTree.py -> build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> copying Cython/Shadow.py -> build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> copying Cython/Debugging.py -> build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> copying Cython/Coverage.py -> build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> copying Cython/CodeWriter.py -> build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Build
python3.13t-cython> copying Cython/Build/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build
python3.13t-cython> copying Cython/Build/IpythonMagic.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build
python3.13t-cython> copying Cython/Build/Inline.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build
python3.13t-cython> copying Cython/Build/Distutils.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build
python3.13t-cython> copying Cython/Build/Dependencies.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build
python3.13t-cython> copying Cython/Build/Cythonize.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build
python3.13t-cython> copying Cython/Build/BuildExecutable.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Visitor.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Version.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/UtilityCode.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/UtilNodes.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/UFuncs.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/TypeSlots.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/TypeInference.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/TreePath.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/TreeFragment.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Symtab.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/StringEncoding.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Scanning.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Pythran.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/PyrexTypes.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Pipeline.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Parsing.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/ParseTreeTransforms.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Options.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Optimize.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Nodes.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Naming.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/ModuleNode.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/MemoryView.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Main.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Lexicon.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Interpreter.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Future.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/FusedNode.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/FlowControl.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/ExprNodes.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Errors.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/DebugFlags.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Dataclass.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/CythonScope.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/CodeGeneration.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Code.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/CmdLine.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Builtin.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Buffer.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/AutoDocTransforms.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Annotate.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/AnalysedTreeTransforms.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Runtime
python3.13t-cython> copying Cython/Runtime/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Runtime
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Distutils
python3.13t-cython> copying Cython/Distutils/old_build_ext.py -> build/lib.linux-x86_64-cpython-313t/Cython/Distutils
python3.13t-cython> copying Cython/Distutils/extension.py -> build/lib.linux-x86_64-cpython-313t/Cython/Distutils
python3.13t-cython> copying Cython/Distutils/build_ext.py -> build/lib.linux-x86_64-cpython-313t/Cython/Distutils
python3.13t-cython> copying Cython/Distutils/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Distutils
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Debugger
python3.13t-cython> copying Cython/Debugger/libpython.py -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger
python3.13t-cython> copying Cython/Debugger/libcython.py -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger
python3.13t-cython> copying Cython/Debugger/DebugWriter.py -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger
python3.13t-cython> copying Cython/Debugger/Cygdb.py -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger
python3.13t-cython> copying Cython/Debugger/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Debugger/Tests
python3.13t-cython> copying Cython/Debugger/Tests/test_libpython_in_gdb.py -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger/Tests
python3.13t-cython> copying Cython/Debugger/Tests/test_libcython_in_gdb.py -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger/Tests
python3.13t-cython> copying Cython/Debugger/Tests/TestLibCython.py -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger/Tests
python3.13t-cython> copying Cython/Debugger/Tests/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger/Tests
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Transitions.py -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Scanners.py -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Regexps.py -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Machines.py -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Lexicons.py -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Errors.py -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/DFA.py -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Actions.py -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Tests
python3.13t-cython> copying Cython/Tests/xmlrunner.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tests
python3.13t-cython> copying Cython/Tests/TestTestUtils.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tests
python3.13t-cython> copying Cython/Tests/TestStringIOTree.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tests
python3.13t-cython> copying Cython/Tests/TestShadow.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tests
python3.13t-cython> copying Cython/Tests/TestJediTyper.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tests
python3.13t-cython> copying Cython/Tests/TestCythonUtils.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tests
python3.13t-cython> copying Cython/Tests/TestCodeWriter.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tests
python3.13t-cython> copying Cython/Tests/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tests
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Build/Tests
python3.13t-cython> copying Cython/Build/Tests/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build/Tests
python3.13t-cython> copying Cython/Build/Tests/TestStripLiterals.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build/Tests
python3.13t-cython> copying Cython/Build/Tests/TestRecythonize.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build/Tests
python3.13t-cython> copying Cython/Build/Tests/TestIpythonMagic.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build/Tests
python3.13t-cython> copying Cython/Build/Tests/TestInline.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build/Tests
python3.13t-cython> copying Cython/Build/Tests/TestDependencies.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build/Tests
python3.13t-cython> copying Cython/Build/Tests/TestCythonizeArgsParser.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build/Tests
python3.13t-cython> copying Cython/Build/Tests/TestCyCache.py -> build/lib.linux-x86_64-cpython-313t/Cython/Build/Tests
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/Utils.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestVisitor.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestUtilityLoad.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestTypes.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestTreePath.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestTreeFragment.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestStringEncoding.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestSignatureMatching.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestScanning.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestParseTreeTransforms.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestMemView.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestGrammar.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestFlowControl.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestCmdLine.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/TestBuffer.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> copying Cython/Compiler/Tests/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler/Tests
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Dataclasses.py -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Tempita
python3.13t-cython> copying Cython/Tempita/compat3.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tempita
python3.13t-cython> copying Cython/Tempita/_tempita.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tempita
python3.13t-cython> copying Cython/Tempita/_looper.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tempita
python3.13t-cython> copying Cython/Tempita/__init__.py -> build/lib.linux-x86_64-cpython-313t/Cython/Tempita
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/pyximport
python3.13t-cython> copying pyximport/pyximport.py -> build/lib.linux-x86_64-cpython-313t/pyximport
python3.13t-cython> copying pyximport/pyxbuild.py -> build/lib.linux-x86_64-cpython-313t/pyximport
python3.13t-cython> copying pyximport/_pyximport3.py -> build/lib.linux-x86_64-cpython-313t/pyximport
python3.13t-cython> copying pyximport/_pyximport2.py -> build/lib.linux-x86_64-cpython-313t/pyximport
python3.13t-cython> copying pyximport/__init__.py -> build/lib.linux-x86_64-cpython-313t/pyximport
python3.13t-cython> copying Cython/Shadow.pyi -> build/lib.linux-x86_64-cpython-313t/Cython
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Includes
python3.13t-cython> copying Cython/Includes/openmp.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/wait.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/unistd.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/uio.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/types.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/time.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/strings.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/stdlib.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/stdio.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/stat.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/signal.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/select.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/resource.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/mman.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/ioctl.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/fcntl.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/dlfcn.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> copying Cython/Includes/posix/__init__.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/posix
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Includes/numpy
python3.13t-cython> copying Cython/Includes/numpy/math.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/numpy
python3.13t-cython> copying Cython/Includes/numpy/__init__.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/numpy
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/vector.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/utility.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/unordered_set.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/unordered_map.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/typeinfo.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/typeindex.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/string.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/stack.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/set.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/random.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/queue.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/pair.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/optional.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/numeric.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/numbers.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/memory.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/map.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/list.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/limits.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/iterator.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/functional.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/forward_list.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/execution.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/deque.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/complex.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/cmath.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/cast.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/bit.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/atomic.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/any.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/algorithm.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> copying Cython/Includes/libcpp/__init__.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libcpp
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/time.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/string.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/stdlib.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/stdio.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/stdint.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/stddef.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/signal.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/setjmp.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/math.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/locale.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/limits.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/float.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/errno.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/complex.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> copying Cython/Includes/libc/__init__.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/libc
python3.13t-cython> creating build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/weakref.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/version.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/unicode.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/type.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/tuple.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/time.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/string.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/slice.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/set.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/sequence.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/ref.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/pythread.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/pystate.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/pyport.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/pylifecycle.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/pycapsule.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/oldbuffer.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/object.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/number.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/module.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/method.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/memoryview.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/mem.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/marshal.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/mapping.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/longintrepr.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/long.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/list.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/iterobject.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/iterator.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/int.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/instance.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/getargs.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/genobject.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/function.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/float.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/fileobject.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/exc.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/dict.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/descr.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/datetime.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/conversion.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/contextvars.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/complex.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/codecs.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/cobject.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/ceval.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/cellobject.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/bytes.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/bytearray.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/buffer.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/bool.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/array.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Includes/cpython/__init__.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Includes/cpython
python3.13t-cython> copying Cython/Compiler/Visitor.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Scanning.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Parsing.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/ParseTreeTransforms.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/FlowControl.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Compiler/Code.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Compiler
python3.13t-cython> copying Cython/Runtime/refnanny.pyx -> build/lib.linux-x86_64-cpython-313t/Cython/Runtime
python3.13t-cython> copying Cython/Debugger/Tests/codefile -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger/Tests
python3.13t-cython> copying Cython/Debugger/Tests/cfuncs.c -> build/lib.linux-x86_64-cpython-313t/Cython/Debugger/Tests
python3.13t-cython> copying Cython/Plex/Transitions.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Scanners.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Machines.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/DFA.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Plex/Actions.pxd -> build/lib.linux-x86_64-cpython-313t/Cython/Plex
python3.13t-cython> copying Cython/Utility/UFuncs.pyx -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/TestCythonScope.pyx -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/TestCyUtilityLoader.pyx -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/MemoryView.pyx -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/CppConvert.pyx -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/CpdefEnums.pyx -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/CConvert.pyx -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/UFuncs_C.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/TypeConversion.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/TestUtilityLoader.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/StringTools.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Profile.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Printing.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Overflow.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Optimize.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/ObjectHandling.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/NumpyImportArray.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/ModuleSetupCode.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/MemoryView_C.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/ImportExport.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/FunctionArguments.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/ExtensionTypes.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Exceptions.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Embed.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Dataclasses.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/CythonFunction.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Coroutine.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Complex.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/CommonStructures.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/CMath.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Builtins.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/Buffer.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/AsyncGen.c -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/arrayarray.h -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> copying Cython/Utility/CppSupport.cpp -> build/lib.linux-x86_64-cpython-313t/Cython/Utility
python3.13t-cython> running build_ext
python3.13t-cython> Compiling /build/source/Cython/Compiler/Parsing.py because it changed.
python3.13t-cython> [1/1] Cythonizing /build/source/Cython/Compiler/Parsing.py
python3.13t-cython> building 'Cython.Compiler.Parsing' extension
python3.13t-cython> creating build/temp.linux-x86_64-cpython-313t/build/source/Cython/Compiler
python3.13t-cython> gcc -fno-strict-overflow -Wsign-compare -DNDEBUG -g -O3 -Wall -I/nix/store/qizipyz9y17nr4w4gmxvwd3x4k0bp2rh-libxcrypt-4.4.38/include -fPIC -I/nix/store/ndvcyyrxq3dfw4zc62pkzqfhi4qhc37n-python3-3.13.3/include/python3.13t -c /build/source/Cython/Compiler/Parsing.c -o build/temp.linux-x86_64-cpython-313t/build/source/Cython/Compiler/Parsing.o
python3.13t-cython> /nix/store/xy4jjgw87sbgwylm5kn047d9gkbhsr9x-bash-5.2p37/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8): No such file or directory
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c:3088:80: error: unknown type name '__pyx_vectorcallfunc'; did you mean 'vectorcallfunc'?
python3.13t-cython>  3088 | static CYTHON_INLINE PyObject *__Pyx_PyVectorcall_FastCallDict(PyObject *func, __pyx_vectorcallfunc vc, PyObject *const *args, size_t nargs, PyObject *kw);
python3.13t-cython>       |                                                                                ^~~~~~~~~~~~~~~~~~~~
python3.13t-cython>       |                                                                                vectorcallfunc
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c: In function '__pyx_f_6Cython_8Compiler_7Parsing_p_dict_or_set_maker':
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c:32216:9: warning: 'Py_OptimizeFlag' is deprecated [-Wdeprecated-declarations]
python3.13t-cython> 32216 |         if (unlikely(__pyx_assertions_enabled())) {
python3.13t-cython>       |         ^~
python3.13t-cython> In file included from /nix/store/ndvcyyrxq3dfw4zc62pkzqfhi4qhc37n-python3-3.13.3/include/python3.13t/Python.h:76,
python3.13t-cython>                  from /build/source/Cython/Compiler/Parsing.c:31:
python3.13t-cython> /nix/store/ndvcyyrxq3dfw4zc62pkzqfhi4qhc37n-python3-3.13.3/include/python3.13t/cpython/pydebug.h:13:37: note: declared here
python3.13t-cython>    13 | Py_DEPRECATED(3.12) PyAPI_DATA(int) Py_OptimizeFlag;
python3.13t-cython>       |                                     ^~~~~~~~~~~~~~~
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c: At top level:
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c:90455:69: error: unknown type name '__pyx_vectorcallfunc'; did you mean 'vectorcallfunc'?
python3.13t-cython> 90455 | static PyObject *__Pyx_PyVectorcall_FastCallDict_kw(PyObject *func, __pyx_vectorcallfunc vc, PyObject *const *args, size_t nargs, PyObject *kw)
python3.13t-cython>       |                                                                     ^~~~~~~~~~~~~~~~~~~~
python3.13t-cython>       |                                                                     vectorcallfunc
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c:90500:80: error: unknown type name '__pyx_vectorcallfunc'; did you mean 'vectorcallfunc'?
python3.13t-cython> 90500 | static CYTHON_INLINE PyObject *__Pyx_PyVectorcall_FastCallDict(PyObject *func, __pyx_vectorcallfunc vc, PyObject *const *args, size_t nargs, PyObject *kw)
python3.13t-cython>       |                                                                                ^~~~~~~~~~~~~~~~~~~~
python3.13t-cython>       |                                                                                vectorcallfunc
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c: In function '__Pyx_CyFunction_CallAsMethod':
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c:91189:6: error: unknown type name '__pyx_vectorcallfunc'; did you mean 'vectorcallfunc'?
python3.13t-cython> 91189 |      __pyx_vectorcallfunc vc = __Pyx_CyFunction_func_vectorcall(cyfunc);
python3.13t-cython>       |      ^~~~~~~~~~~~~~~~~~~~
python3.13t-cython>       |      vectorcallfunc
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c:3178:45: error: initialization of 'int' from 'vectorcallfunc' {aka 'struct _object * (*)(struct _object *, struct _object * const*, long unsigned int,  struct _object *)'} makes integer from pointer without a cast [-Wint-conversion]
python3.13t-cython>  3178 | #define __Pyx_CyFunction_func_vectorcall(f) (((PyCFunctionObject*)f)->vectorcall)
python3.13t-cython>       |                                             ^
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c:91189:32: note: in expansion of macro '__Pyx_CyFunction_func_vectorcall'
python3.13t-cython> 91189 |      __pyx_vectorcallfunc vc = __Pyx_CyFunction_func_vectorcall(cyfunc);
python3.13t-cython>       |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c:91192:16: error: implicit declaration of function '__Pyx_PyVectorcall_FastCallDict'; did you mean '__Pyx_PyObject_FastCallDict'? [-Wimplicit-function-declaration]
python3.13t-cython> 91192 |         return __Pyx_PyVectorcall_FastCallDict(func, vc, &PyTuple_GET_ITEM(args, 0), (size_t)PyTuple_GET_SIZE(args), kw);
python3.13t-cython>       |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
python3.13t-cython>       |                __Pyx_PyObject_FastCallDict
python3.13t-cython> /build/source/Cython/Compiler/Parsing.c:91192:16: error: returning 'int' from a function with return type 'PyObject *' {aka 'struct _object *'} makes pointer from integer without a cast [-Wint-conversion]
python3.13t-cython> 91192 |         return __Pyx_PyVectorcall_FastCallDict(func, vc, &PyTuple_GET_ITEM(args, 0), (size_t)PyTuple_GET_SIZE(args), kw);
python3.13t-cython>       |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
python3.13t-cython> error: command '/nix/store/0fsnicvfpf55nkza12cjnad0w84d6ba7-gcc-wrapper-14.2.1.20250322/bin/gcc' failed with exit code 1
python3.13t-cython>
python3.13t-cython> ERROR Backend subprocess exited when trying to invoke build_wheel
error: Cannot build '/nix/store/m9xm38wch7fjrmwvia27nl60xdzm3j98-python3.13t-cython-3.0.12.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/n700cjqm7gy5x86rj242qxnp0p3lmy33-python3.13t-cython-3.0.12-dist
         /nix/store/sa5h95yl2maw80j7hr39mdw5xsxqgvpd-python3.13t-cython-3.0.12
       Last 25 log lines:
       > /build/source/Cython/Compiler/Parsing.c:90500:80: error: unknown type name '__pyx_vectorcallfunc'; did you mean 'vectorcallfunc'?
       > 90500 | static CYTHON_INLINE PyObject *__Pyx_PyVectorcall_FastCallDict(PyObject *func, __pyx_vectorcallfunc vc, PyObject *const *args, size_t nargs, PyObject *kw)
       >       |                                                                                ^~~~~~~~~~~~~~~~~~~~
       >       |                                                                                vectorcallfunc
       > /build/source/Cython/Compiler/Parsing.c: In function '__Pyx_CyFunction_CallAsMethod':
       > /build/source/Cython/Compiler/Parsing.c:91189:6: error: unknown type name '__pyx_vectorcallfunc'; did you mean 'vectorcallfunc'?
       > 91189 |      __pyx_vectorcallfunc vc = __Pyx_CyFunction_func_vectorcall(cyfunc);
       >       |      ^~~~~~~~~~~~~~~~~~~~
       >       |      vectorcallfunc
       > /build/source/Cython/Compiler/Parsing.c:3178:45: error: initialization of 'int' from 'vectorcallfunc' {aka 'struct _object * (*)(struct _object *, struct _object * const*, long unsigned int,  struct _object *)'} makes integer from pointer without a cast [-Wint-conversion]
       >  3178 | #define __Pyx_CyFunction_func_vectorcall(f) (((PyCFunctionObject*)f)->vectorcall)
       >       |                                             ^
       > /build/source/Cython/Compiler/Parsing.c:91189:32: note: in expansion of macro '__Pyx_CyFunction_func_vectorcall'
       > 91189 |      __pyx_vectorcallfunc vc = __Pyx_CyFunction_func_vectorcall(cyfunc);
       >       |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > /build/source/Cython/Compiler/Parsing.c:91192:16: error: implicit declaration of function '__Pyx_PyVectorcall_FastCallDict'; did you mean '__Pyx_PyObject_FastCallDict'? [-Wimplicit-function-declaration]
       > 91192 |         return __Pyx_PyVectorcall_FastCallDict(func, vc, &PyTuple_GET_ITEM(args, 0), (size_t)PyTuple_GET_SIZE(args), kw);
       >       |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       >       |                __Pyx_PyObject_FastCallDict
       > /build/source/Cython/Compiler/Parsing.c:91192:16: error: returning 'int' from a function with return type 'PyObject *' {aka 'struct _object *'} makes pointer from integer without a cast [-Wint-conversion]
       > 91192 |         return __Pyx_PyVectorcall_FastCallDict(func, vc, &PyTuple_GET_ITEM(args, 0), (size_t)PyTuple_GET_SIZE(args), kw);
       >       |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > error: command '/nix/store/0fsnicvfpf55nkza12cjnad0w84d6ba7-gcc-wrapper-14.2.1.20250322/bin/gcc' failed with exit code 1
       >
       > ERROR Backend subprocess exited when trying to invoke build_wheel
       For full logs, run:
         nix log /nix/store/m9xm38wch7fjrmwvia27nl60xdzm3j98-python3.13t-cython-3.0.12.drv
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.14.9-zen1, NixOS, 25.11 (Xantusia), 25.11.20250610.f858d0a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Determinate Nix 3.6.2) 2.29.0`
 - nixpkgs: `/nix/store/qvwi35a98y2i4qw3iplaixd7mxn0xmmj-source`

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/aiohttp/default.nix
+++ b/pkgs/development/python-modules/aiohttp/default.nix
@@ -8,7 +8,7 @@
   isPyPy,
 
   # build-system
-  cython_3_1,
+  cython,
   pkgconfig,
   setuptools,
 
@@ -75,7 +75,7 @@ buildPythonPackage rec {
   '';
 
   build-system = [
-    cython_3_1
+    cython
     pkgconfig
     setuptools
   ];

--- a/pkgs/development/python-modules/cython/default.nix
+++ b/pkgs/development/python-modules/cython/default.nix
@@ -16,14 +16,14 @@
 
 buildPythonPackage rec {
   pname = "cython";
-  version = "3.0.12";
+  version = "3.1.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "cython";
     repo = "cython";
     tag = version;
-    hash = "sha256-clJXjQb6rVECirKRUGX0vD5a6LILzPwNo7+6KKYs2pI=";
+    hash = "sha256-lP8ILCzAZuoPzFhCqGXwIpifN8XoWz93SJ7c3XVe69Y=";
   };
 
   build-system = [

--- a/pkgs/development/python-modules/pyyaml-ft/default.nix
+++ b/pkgs/development/python-modules/pyyaml-ft/default.nix
@@ -1,6 +1,6 @@
 {
   buildPythonPackage,
-  cython_3_1,
+  cython,
   fetchFromGitHub,
   lib,
   libyaml,
@@ -24,7 +24,7 @@ buildPythonPackage rec {
   };
 
   build-system = [
-    cython_3_1
+    cython
     setuptools
   ];
 

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -3176,16 +3176,6 @@ self: super: with self; {
 
   cython_0 = callPackage ../development/python-modules/cython/0.nix { };
 
-  cython_3_1 = cython.overridePythonAttrs rec {
-    version = "3.1.2";
-    src = pkgs.fetchFromGitHub {
-      owner = "cython";
-      repo = "cython";
-      tag = version;
-      hash = "sha256-lP8ILCzAZuoPzFhCqGXwIpifN8XoWz93SJ7c3XVe69Y=";
-    };
-  };
-
   cytoolz = callPackage ../development/python-modules/cytoolz { };
 
   dacite = callPackage ../development/python-modules/dacite { };

